packages/core/crates/core-strategy/Cargo.toml

```diff
@@ -13,7 +13,7 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 async-trait = "0.1"
-async-std = "1.12"
+async-std = { version = "1.12.0", features = ["unstable"]}
```

Comment:
Please use `workspace = true` instead of version

Reply:
updated

Reply:
```suggestion
async-std = { workspace = true, features = ["unstable"] }
```

Reply:
```suggestion
async-std = { workspace = true, features = ["unstable"] }
```

Reply:
```suggestion
async-std = { workspace = true, features = ["unstable"] }
```

Reply:
```suggestion
async-std = { workspace = true, features = ["unstable"] }
```